Dockerfile

```
@@ -1,5 +1,5 @@
 # NOTE(dperny): for some reason, alpine was giving me trouble
-ARG GO_VERSION=1.13.15
+ARG GO_VERSION=1.17.2
```

😅 looks like we're slightly behind (sorry, didn't notice before)

```suggestion
ARG GO_VERSION=1.17.4
```

---

When updating; perhaps split it into two commits;

- squash the "Avoid boolean misinterpretation" and Go 1.13 -> 1.17 (only go update)
- (slightly cleaner history) move the `ioutil -> os` changes to a separate commit (similar to https://github.com/moby/moby/pull/42792)